packages/voting/manifest/data/kovan.json

```diff
@@ -1,9 +1,12 @@
 {
   "network": "kovan",
-  "votingTokenAddress": "0x489Bf230d4Ab5c2083556E394a28276C22c3B580",
```

Comment:
why remove this?

Reply:
nvm I guess it's in the `token` package

Reply:
We don't use it anywhere, its hardcoded in `utils/constants`